### ISSUE TITLE
uninitialized constant Puma::MiniSSL::SSLError (NameError)

### ISSUE THREAD

**Issue by amedeiros** [puma/puma#745]
I keep seeing the following error in a small sinatra application from puma.

```
Error in reactor loop escaped: uninitialized constant Puma::MiniSSL::SSLError (NameError)
org/jruby/RubyModule.java:2935:in `const_missing'
/Users/amedeiros/.rvm/gems/jruby-9.0.0.0.pre1@esp-query/gems/puma-2.12.2-java/lib/puma/reactor.rb:72:in `run_internal'
org/jruby/RubyArray.java:1569:in `each'
/Users/amedeiros/.rvm/gems/jruby-9.0.0.0.pre1@esp-query/gems/puma-2.12.2-java/lib/puma/reactor.rb:41:in `run_internal'
/Users/amedeiros/.rvm/gems/jruby-9.0.0.0.pre1@esp-query/gems/puma-2.12.2-java/lib/puma/reactor.rb:137:in `run_in_thread'

```

**Comment by gregors** [comment]
seeing the same

**Comment by deivid-rodriguez** [comment]
Guys, this was fixed in https://github.com/puma/puma/pull/742. The warning should be gone in the next release, so you can close this!

**Comment by dqdinh** [comment]
Does anyone know where I can find when the next release will be out?

**Comment by evanphx** [comment]
I'll do one this evening.

**Comment by evanphx** [comment]
This is fixed in 2.12.3.
